src/views/Game/__snapshots__/Game.test.tsx.snap

```diff
@@ -1,7014 +1,994 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Game should match snapshot 1`] = `
-ShallowWrapper {
-  Symbol(enzyme.__root__): [Circular],
-  Symbol(enzyme.__unrendered__): <Game
-    activeBoards={Array []}
-    board={
+<div
+  className="center"
+>
+  <div
+    className="gameWrapper"
+  >
+    <ConnectFunction />
+    <BigBoard
```

Comment:
Reference to the other comment: Here there is the BigBoard represented as a child but not in the other snapshot 😲

Reply:
Not actually a problem with the snapshots as commented above.

Reply:
Not actually a problem with the snapshots as commented above.